RELEASES.md

```diff
@@ -0,0 +1,87 @@
+# Releases
+
+## Versioning
+
+SOCI versions follows the `Major.Minor.Patch-PreRelease` scheme of [Semantic Versioning](https://semver.org).
+
+### `soci CLI / soci-snapshotter / SOCI library`
```

Comment:
The use of `formatting` here and below is a bit confusing; some of these words are commands/binaries and others just descriptors.

Reply:
Does extrapolating @djdongjin's comments below work for you? i.e. only using the formatting for binary names?

Reply:
yes, or possibly also command line parameters if that becomes relevant.